Formula/chuck.rb

```diff
@@ -13,6 +13,8 @@ class Chuck < Formula
 
   depends_on :xcode => :build
 
+  patch :DATA
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!